code/src/test/java/frc/robot/mocks/subsystems/MockDrivetrain.java

```diff
@@ -136,7 +137,7 @@ public void moveEast(double maxSpeed, double distance) {
     }
 
     @Override
-    public void moveWest(double distance) {
+    public void moveWest(double distance) {//do
         this.faceWest();
```

Comment:
We should discuss if the moveDirection methods rotate the robot. I was under the impression that these methods were to move the robot in a field relative view. I.E. this method would move the robot to driver west, not caring about which direction the robot is facing.